pochivm/api_base.h

```diff
@@ -117,111 +117,223 @@ Value<T>::operator Value<U>() const
 
 // Arithmetic ops convenience operator overloading
 //
```

Comment:
move this comment to the right place, and add comment on what your 'DoArith' is doing